pkg/api/v1/router.go

```diff
@@ -60,4 +63,11 @@ func (r *Router) Routes(e *echo.Echo) {
 		r.addOriginRoutes(v1)
 		r.addPoolsRoutes(v1)
 	}
+
+	r.events.AddStream(&nats.StreamConfig{
+		Name: "lbapi",
+		Subjects: []string{
+			"com.infratographer.events.>",
```

Comment:
same? or use the exported prefix?

Reply:
I think the above will help answer this^